fitnessllm_dataplatform/utils/cloud_utils.py

```
@@ -0,0 +1,32 @@
+def create_resource_path(project_id: str, service: str, name: str) -> str:
+    """Simple builder for resource paths."""
+    return f"projects/{project_id}/{service}/{name}/versions/latest"
+
+
+def get_secret(name: str) -> dict:
+    """Retrieve secret from secret manager."""
+    logger.debug("Initializing secret manager")
+    client = secretmanager.SecretManagerServiceClient()
+    logger.debug(f"Getting secret for {name}")
+    response = client.access_secret_version(
+        request={"name": create_resource_path(environ["PROJECT_ID"], "secrets", name)}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)